pyserini/search/faiss/_searcher.py

```diff
@@ -477,6 +479,14 @@ def search(self, query: Union[str, np.ndarray], k: int = 10, threads: int = 1, r
             distances, indexes = self.index.search(emb_q, k)
             distances = distances.flat
             indexes = indexes.flat
+            if remove_dups == True:
```

Comment:
```suggestion
            if remove_dups:
```

I think you can just do this?

Please re-verify tests.

Reply:
This works! Thanks for the advice